Assets/Scripts/Battlefield/PlayerInputHandler.cs

```diff
@@ -1,23 +1,28 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using SwordAndBored.Battlefield.CreaturScripts;
+using SwordAndBored.Battlefield.TurnMechanism;
+using SwordAndBored.Battlefield.CameraUtilities;
 
-
-namespace SwordAndBored.BattleMechanism
+namespace SwordAndBored.Battlefield
 {
-    public class Interact : MonoBehaviour
+    public class PlayerInputHandler : MonoBehaviour
```

Comment:
should probably be integrated into the "TurnBrain" so that a player turnbrain subclass has this and an enemy turnbrain subclass doesn't and this will also ensure that every unit has its own instead of a master InputHandler that has to be passed around to all player units. Performance-wise I think we can take the hit without worrying too much for now since there will never be more than 6 or instances of these per battlefield.